pkg/scheduler/algorithm/priorities/image_locality.go

```diff
@@ -34,6 +34,10 @@ const (
 	maxThreshold int64 = 1000 * mb
 )
 
+// alpha is used to fine-tune the image score scaling
+// TODO: expose the alpha parameter to config map
+const alpha float64 = 0
```

Comment:
I think `alpha` is not a self-explained name, especially if you want expose it as a flag in the future. Mind to re-think of another one?

Reply:
Yes, I agree - as mentioned in the special notes, this can be seen as WIP and not urgent for 1.12 :) It could've been an issue not a PR though I thought with a PR it was clearer to explain.

How about "trueLocalityRatio" - a ratio values [0, 1] and the higher the ratio the higher we take into account the "true locality"/locality without scaling heuristics.